docs/sources/reference/api/docker_remote_api_v1.19.md

```diff
@@ -109,131 +109,130 @@ Create a container
-        }
+    POST /containers/create HTTP/1.1
+    Content-Type: application/json
+
+    {
+           "Hostname": "",
+           "Domainname": "",
+           "User": "",
+           "AttachStdin": false,
+           "AttachStdout": true,
+           "AttachStderr": true,
+           "Tty": false,
+           "OpenStdin": false,
+           "StdinOnce": false,
+           "Env": null,
+           "Cmd": [
+                   "date"
+           ],
+           "Entrypoint": "",
+           "Image": "ubuntu",
+           "Labels": {
+                   "com.example.vendor": "Acme",
+                   "com.example.license": "GPL",
+                   "com.example.version": "1.0"
+           },
+           "Volumes": {
+                   "/tmp": {}
+           },
+           "WorkingDir": "",
+           "NetworkDisabled": false,
+           "MacAddress": "12:34:56:78:9a:bc",
+           "ExposedPorts": {
+                   "22/tcp": {}
+           },
+           "HostConfig": {
+             "Binds": ["/tmp:/tmp"],
+             "Links": ["redis3:redis"],
+             "LxcConf": {"lxc.utsname":"docker"},
+             "Memory": 0,
+             "MemorySwap": 0,
+             "CpuShares": 512,
+             "CpuPeriod": 100000,
+             "CpusetCpus": "0,1",
+             "CpusetMems": "0,1",
+             "BlkioWeight": 300,
+             "OomKillDisable": false,
+             "PortBindings": { "22/tcp": [{ "HostPort": "11022" }] },
+             "PublishAllPorts": false,
+             "Privileged": false,
+             "ReadonlyRootfs": false,
+             "Dns": ["8.8.8.8"],
+             "DnsSearch": [""],
+             "ExtraHosts": null,
+             "VolumesFrom": ["parent", "other:ro"],
+             "CapAdd": ["NET_ADMIN"],
+             "CapDrop": ["MKNOD"],
+             "RestartPolicy": { "Name": "", "MaximumRetryCount": 0 },
+             "NetworkMode": "bridge",
+             "Devices": [],
+             "Ulimits": [{}],
+             "LogConfig": { "Type": "json-file", "Config": {} },
+             "SecurityOpt": [""],
+             "CgroupParent": ""
+          }
+      }
 
 **Example response**:
 
-        HTTP/1.1 201 Created
-        Content-Type: application/json
+      HTTP/1.1 201 Created
+      Content-Type: application/json
 
-        {
-             "Id":"e90e34656806"
-             "Warnings":[]
-        }
+      {
+           "Id":"e90e34656806"
+           "Warnings":[]
+      }
 
 Json Parameters:
 
--   **Hostname** - A string value containing the desired hostname to use for the
+-   **Hostname** - A string value containing the hostname to use for the
       container.
--   **Domainname** - A string value containing the desired domain name to use
+-   **Domainname** - A string value containing the domain name to use
       for the container.
--   **User** - A string value containing the user to use inside the container.
+-   **User** - A string value specifying the user to use inside the container.
```

Comment:
"the user to use"

need something better at some point as well, no (short) suggestion at this moment. It's the user that the process inside the container will be run as.